### PR TITLE
fix(aio): correctly process markdown link in "Browser Support"

### LINKED ISSUE #18312: Broken link in Browser Support guide
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

On [the browser support page](https://angular.io/guide/browser-support#optional-browser-features-to-polyfill) the Http entry for optional browser features has a broken link. Maybe the Http-Client addition modified some routes and caused this to break?

## Expected behavior

The Http entry for browser features that can be polyfilled should have a correct API reference/guide.

## Minimal reproduction of the problem with instructions

Go to https://angular.io/guide/browser-support#optional-browser-features-to-polyfill and look at the last entry in the table.

## What is the motivation / use case for changing the behavior?

Accurate docs.

## Environment

<pre><code>
Angular version: 4.3.1

Browser:
- [x] Chrome (desktop) version latest
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  
n/a

Others:

</code></pre>
n/a

### PR DESCRIPTION
**Pull request by gkalpak**
The markdown processor expects an empty line between an opening tag and the markdown content. (If there is no empty line, the content is interpreted as plain HTML.)

Previously, the line between the opening `<td>` and the content contained whitespace, which caused the content to be interpreted as HTML and not markdown.

Fixes #18312.

### CODE REVIEW

File: aio/content/guide/browser-support.md [hunk 1]
```diff
@@ -382,16 +382,17 @@ Here are the features which may require additional polyfills:
   </tr>
 
   <tr style="vertical-align: top">
- 
+
     <td>
-
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md]: This was causing the issue (the other whitespace changes are unrelated).

**Review by petebacondarwin — APPROVED** [review]

**Comment by gkalpak** [comment]
@petebacondarwin, could we pre-preprocess the guides (before passing to remark) to remove all trailing whitespace? (Trailing whitespace might be important in markdown, but leaving markdown content unprocessed is more problematic than missing a newline.)

**Comment by petebacondarwin** [comment]
How about we have a content lint rule that checks for trailing whitespace
our guide files?

**Comment by gkalpak** [comment]
Works for me as well :smiley:
